docker/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ ! -f /cassy/data/cassy.db ]; then
+  sqlite3 /cassy/data/cassy.db < /cassy/scripts/db.schema
+fi
+
+exec "$@"
```

Comment:
Is it needed ?

Reply:
Yes, because `CMD` elements are passed to `ENTRYPOINT` as arguments. `exec "$@"` is a common pattern for entrypoint.